processing.js

```diff
@@ -2497,7 +2497,7 @@ module.exports = function(options, undef) {
      * @param {float} sy  the amount to scale on the y-axis
      */
     scale: function(sx, sy) {
-      if (sx && !sy) {
+      if (sx && sy === undef) {
```

Comment:
I wonder why you've switched the elegant `!sy` by the ugly `sy === undef`.
And here's a danger: `sy === undef` doesn't catch `null`. Therefore code transpiled from Java Mode to JS Mode is gonna FAIL!